frontend/www/js/omegaup/components/user/Profilev2.vue

```diff
@@ -240,22 +240,24 @@ import { Problem, ContestResult } from '../../linkable_resource';
   },
 })
 export default class UserProfile extends Vue {
-  @Prop() data!: types.ExtraProfileDetails;
+  @Prop() data!: types.ExtraProfileDetails | null;
   @Prop() profile!: types.UserProfileInfo;
   @Prop() profileBadges!: Set<string>;
   @Prop() visitorBadges!: Set<string>;
-  contests = this.data.contests
-    ? Object.values(this.data.contests)
-        .map((contest) => {
-          const now = new Date();
-          if (contest.place === null || now <= contest.data.finish_time) {
-            return null;
-          }
-          return new ContestResult(contest);
-        })
-        .filter((contest) => !!contest)
+  contests = this.data
+    ? this.data.contests
+      ? Object.values(this.data.contests)
+          .map((contest) => {
+            const now = new Date();
+            if (contest.place === null || now <= contest.data.finish_time) {
+              return null;
+            }
+            return new ContestResult(contest);
+          })
+          .filter((contest) => !!contest)
+      : []
     : [];
```

Comment:
se puede mover esto a una función? más de un ternario es difícil de leer.

alternativamente

```suggestion
  contests = Object.values(this.data?.contests ?? ({} as types.USerProfileContests))
          .map((contest) => {
            const now = new Date();
            if (contest.place === null || now <= contest.data.finish_time) {
              return null;
            }
            return new ContestResult(contest);
          })
          .filter((contest) => Boolean(contest));
```